src/gluonnlp/metric/masked_accuracy.py

```diff
@@ -249,7 +249,10 @@ def update(self, labels, preds, masks=None):
 
         for label, pred_label, mask in zip(labels, preds, masks):
             if pred_label.shape != label.shape:
-                pred_label = ndarray.argmax(pred_label, axis=self.axis)
+                # TODO(haibin) topk does not support fp16. Issue tracked at:
+                # https://github.com/apache/incubator-mxnet/issues/14125
```

Comment:
also leave a note that you're using topk due to speed issue in argmax, and link to the issue `https://github.com/apache/incubator-mxnet/issues/11061`